ee/clickhouse/sql/trends/breakdown.py

```diff
@@ -90,13 +90,13 @@
 
 BREAKDOWN_PROP_JOIN_SQL = """
 WHERE e.team_id = %(team_id)s {event_filter} {filters} {parsed_date_from} {parsed_date_to}
-  AND trim(BOTH '\"' FROM JSONExtractRaw(properties, %(key)s)) in (%(values)s) 
+  AND {breakdown_value_expr} in (%(values)s)
   {actions_query}
 """
 
 NONE_BREAKDOWN_PROP_JOIN_SQL = """
```

Comment:
Ah, my bad, I should've deleted the `NONE_*` breakdown sqls, not used anymore. I will do so!